google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java

```diff
@@ -81,17 +81,15 @@ public void trackEvent(@NotNull String eventCategory,
         List<BasicNameValuePair> postData = Lists.newArrayList(analyticsBaseData);
         postData.add(new BasicNameValuePair("tid", analyticsId));
 
-        postData.add(new BasicNameValuePair("cd19", eventCategory));  // Event type
```

Comment:
I see some options to chose, since the requirement is just that the url should be /virtual/eventType/eventName.

1. Have eventType some kind of high-level category like
/virtual/appengine/deploy-success (dot or dash doesn't matter.)
/virtual/cloud-debugger/create-breakpoint

In this case, there is no room to specify that it is from our plugin. That's very inconvenient. The backend team says we could pass our plugin name as metadata, but that still sounds inconvenient.

2. Set com.google.cloud.tools to eventType (we won't use `.getExternalPluginName()`.)
/virtual/com.google.cloud.tools/appengine.deploy

3. Ignore category completely
/virtual/gcloud-intellij/appengine.deploy